release-notes.md

```diff
@@ -1,3 +1,35 @@
+# Rigel-18.1.0
+## New Game plugin ##
+* Issue Type: Task
+* Issue ID: PLAT-23508
+
+A new plugin to managing Game services with a Redis DB and a new service 'userScore'
```

Comment:
more general - Add plugin for game services with new API userScore service connected to Redis